docs/tutorials/create_architecture.md

```diff
@@ -1,221 +1,221 @@
-# 評価準備
+# Preparation for measuring node and path latency
 
-CARET は、評価する際にアーキテクチャファイルの作成が必要です。
+CARET can observe as following:
 
-トレース結果からアーキテクチャファイルの雛形を作成できますが、一部手作業での修正が必要になります。  
-ここでは、アーキテクチャファイルの作成方法とその修正方法を説明します。
+- callback latency
+- communication latency (inter-node)
+- node latency (intra-node)
+- path latency (intra-node & inter-node)
 
-## アーキテクチャファイルとは
+The following figure shows definition of node latency and path latency.
```

Comment:
Insert line breaks between figures and text.
> The following figure shows definition of node latency and path latency.
>
> [figure]

Reply:
@hsgwa  fixed in https://github.com/tier4/CARET_doc/pull/36/commits/25526e73df6e953f6cc1130fec348ec9cffc2963